test/evergreen.yml

```diff
@@ -2168,6 +2168,22 @@ buildvariants:
     - name: unit-test
     - name: fops
     - name: time-shift-sensitivity-test
+    - name: compile-msan
+    - name: make-check-msan-test
+    - name: compile-ubsan
+    - name: ubsan-test
+    - name: linux-directio
+    - name: syscall-linux
+    - name: compile-asan
+    - name: make-check-asan-test
+    - name: checkpoint-filetypes-test
+    - name: unit-test-long
+    - name: spinlock-gcc-test
+    - name: spinlock-pthread-adaptive-test
+    - name: compile-wtperf
+    - name: wtperf-test
+    - name: ftruncate-test
+    - name: long-test
```

Comment:
Is there any reason we can't alphabetize this list? It would help make this file easier to navigate.

Reply:
I think alphabetising the list would make sense for independent tasks. But this list has some dependent tasks say, compile-msan/make-check-msan-test or compile-ubsan/ubsan-test or compile-asan/make-check-asan-test or compile-wtperf/wtperf-test and I prefer it to be coupled.